.env

```diff
@@ -0,0 +1 @@
+PORT=8080
```

Comment:
no biggie for now, but just want to confirm if you're going to introduce another .env file later 🤔  as I thought .env should be in .gitignore yeah?

Reply:
Yeah I'll put add that to `.gitignore` once there are some sensitive information in this file for sure ✌️